### PR TITLE
Update vacuum docs for xiaomi_miio platform.

### PR DESCRIPTION
**Pull request by OGKevin**
## Proposed change

Updated the vacuum documentation for xiaomi_miio platform. 
All the vacuum attributes have been changed to sensors.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54990
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: https://github.com/home-assistant/core/issues/51361

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by rytilahti — APPROVED** [review]
LGTM :+1:

**Review by bieniu — APPROVED** [review]

File: source/_integrations/xiaomi_miio.markdown [hunk 1]
```diff
@@ -1363,41 +1363,31 @@ automation:
           segments: [1, 1]
 ```
 
+### Sensors
+
+Sensor                        | Description                                                                  | Enabled by default               |
+----------------------------- | ---------------------------------------------------------------------------- | -------------------------------- |
+DnD Start                     | The timestamp when the next DnD will start                                   | False                            |
+DnD End                       | The timestamp when the current or next DnD will end                          | False                            |
+Total duration                | The total cleaning duration in seconds                                       | False                            |
```
Comment (frenck) [inline source/_integrations/xiaomi_miio.markdown:1372]: These tables will be hard to read on mobile (too wide) and are not recommended.

Additionally, `True` & `False` in this context is referring to the Python programming language, however, this is end-users documentation. There is no reason to use `True`/`False` in this context.

Comment (OGKevin) [reply source/_integrations/xiaomi_miio.markdown:1372]: What do you recommend replacing the tables with? 

The True and False indicate if the user should manually enable the sensor or not. Shall this not be included in the end-user documentation? That sensor is enabled by default or not. I mean `Enabled by default: true/false` is not really technical, a non-technical person can also understand that true means yes and false means no 🤔 I'm definitely open to suggestions! 

Or are you saying it should be `Yes`/`No`?

Comment (frenck) [reply source/_integrations/xiaomi_miio.markdown:1372]: > What do you recommend replacing the tables with?

A `basic_config` block or, even simpler, just a list.

> Or are you saying it should be Yes/No?

Yep, or, if you use a list to show these devices, add a `*` to them and explain later that those `*` items are not enabled by default.

**Pull request merged**